packages/backend/migrations/20241119125038_add_unique_constraint_to_kid.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+  return knex
+    .raw(
+      // Keep only one active walletAddressKey (the most recent row)
+      `DELETE FROM "walletAddressKeys" w
+       WHERE revoked = false
+       AND ctid NOT IN (
+          SELECT MAX(ctid)
+          FROM "walletAddressKeys"
+          WHERE revoked = false
+            AND kid = w.kid
+          GROUP BY kid, "walletAddressId"
+       )`
```

Comment:
Do we need 
```suggestion
      `DELETE FROM "walletAddressKeys" w
       AND ctid NOT IN (
          SELECT MAX(ctid)
          FROM "walletAddressKeys"
          WHERE revoked = false
          GROUP BY kid, "walletAddressId"
       )`
```
can you double check if this suggestion has the same behaviour? I _think_ it should